app/assets/javascripts/app/views/location_stream.js

```diff
@@ -1,6 +1,6 @@
 // @license magnet:?xt=urn:btih:0b31508aeb0634b347b8270c7bee4d411b5d4109&dn=agpl-3.0.txt AGPL-v3-or-Later
 
-app.views.LocationStream = app.views.Content.extend({
+app.views.LocationStream = app.views.Base.extend({
```

Comment:
Could you elaborate your reasoning behind this? I feel like I am missing some context.